src/p5.dance.js

```diff
@@ -287,30 +291,38 @@ module.exports = class DanceParty {
       var msPerBeat = 60 * 1000 / (this.songMetadata_.bpm * (sprite.dance_speed / 2));
       var msPerFrame = msPerBeat / FRAMES;
       while (sprite.sinceLastFrame > msPerFrame) {
+        const totalFrames = sprite.current_move > 12 ? 12 : 24;
```

Comment:
Two problems here:

1. Should be `>= 12`; our animations are zero-indexed.
2. `current_move` is used to track the last _looping_ move the player set, and to return to it after a one-time move is complete.  Unfortunately we started using it to track the looping move.  We'll want to separate these.